jupyter-js-widgets/src/widget_int.ts

```diff
@@ -660,12 +662,16 @@ class PlayModel extends BoundedIntModel {
     loop() {
         if (this.get('_playing')) {
             var next_value = this.get('value') + this.get('step');
-            if (next_value < this.get('max')) {
+            if (next_value <= this.get('max')) {
                 this.set('value', next_value);
                 window.setTimeout(this.loop.bind(this), this.get('interval'));
             } else {
                 this.set('value', this.get('min'));
```

Comment:
Can we only reset it to the min if we are repeating? Reverting to the min after a straight play through has been frustrating to me, instead of just stopping at the end and freezing the state there.